docs/advanced_topics/api/v2/configuration.md

```diff
@@ -139,6 +139,33 @@ This will make `published_date`, `body`, `feed_image` and a list of
 fields, you must select the `blog.BlogPage` type using the `?type`
 [parameter in the API itself](apiv2_custom_page_fields).
 
+### Adding form fields to the API
+
+If you have a FormBuilder page called `FormPage` and a form field model called `FormField`, this is
+an example of how you would expose the form fields to the API:
+
+```python
+from wagtail.api import APIField
+
+class FormField(AbstractFormField):
+    page = ParentalKey('FormPage', related_name='form_fields', on_delete=models.CASCADE)
+    
+    api_fields = [
```

Comment:
Wondering if we should put these fields onto the ``AbstractFormField`` class itself? if someone adds ``APIField('form_fields')`` to their form model, they are probably looking to add these anyway

Reply:
That's a good idea! I've updated the branch.